cmd/csi-snapshotter/main.go

```diff
@@ -214,7 +215,8 @@ func main() {
 	if !*leaderElection {
 		run(context.TODO())
 	} else {
-		le := leaderelection.NewLeaderElection(kubeClient, leaderElectionLockName, run)
+		lockName := fmt.Sprintf("%s-%s", prefix, strings.Replace(*snapshotterName, "/", "-", -1))
```

Comment:
I think we don't need this prefix at all.  The lockName used in the provisioner does not have a prefix:
https://github.com/kubernetes-csi/external-provisioner/blob/master/cmd/csi-provisioner/csi-provisioner.go#L202

Reply:
We do need the prefix otherwise the name will conflict with other sidecar's locks.  provisioner unfortunately has more backwards compatibility concerns so it's not so easy to fix: https://github.com/kubernetes-csi/external-provisioner/issues/295

Reply:
Sound good.